src/Community.VisualStudio.Toolkit.Shared/DependencyInjectionContainerToolkitPackage.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Threading;
+using System;
+using Community.VisualStudio.Toolkit.Shared.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.VisualStudio.Shell;
+using Task = System.Threading.Tasks.Task;
+
+namespace Community.VisualStudio.Toolkit
+{
+    /// <summary>
+    /// Package that contains a DI service container.
+    /// </summary>
+    /// <typeparam name="TPackage"></typeparam>
+    [ProvideService(typeof(SToolkitServiceProvider<>), IsAsyncQueryable = true)]
+    public class DependencyInjectionContainerToolkitPackage<TPackage> : ToolkitPackage
```

Comment:
Someone can probably come up with a better name for this than I did 😛

Reply:
Maybe just `DIToolkitPackage` or `ToolkitPackageWithDI`?